### ISSUE TITLE
Invalid apiVersion "5.5" [products processor]

### ISSUE THREAD

**Issue by dimasch** [DivanteLtd/vue-storefront-api#183]
https://github.com/DivanteLtd/vue-storefront-api/blob/58a888852a42f731261508840f1e4aab713a50ed/package.json#L62

```
error: uncaughtException: Invalid apiVersion "5.5", expected a function or one of _default, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 2.4, 1.7, 0.90, 6.7, 7.0, 7.x, master date=Fri Feb 08 2019 11:22:53 GMT+0400 (+04), pid=1365, uid=501, gid=20, cwd=/Users/dima/vsf/api, execPath=/usr/local/Cellar/node@8/8.15.0/bin/node, version=v8.15.0, argv=[node, /Users/dima/vsf/api/src, --presets, env,stage-0], rss=121856000, heapTotal=87474176, heapUsed=57530152, external=195671, loadavg=[2.3447265625, 2.6728515625, 2.9140625], uptime=1470328, trace=[column=13, file=/Users/dima/vsf/api/node_modules/elasticsearch/src/lib/utils.js, function=Object.utils.funcEnum, line=285, method=funcEnum, native=false, column=33, file=/Users/dima/vsf/api/node_modules/elasticsearch/src/lib/client.js
```
Related https://github.com/elastic/elasticsearch-js/issues/765

For fix your can use without `^` in `package.json`
`"elasticsearch": "15.2.0",`
\+ @filrak @pkarw 

And don't forget `mage2storefront` https://github.com/DivanteLtd/mage2vuestorefront/blob/159e717de1ec069cb5ca9032b4118229c4d717a5/package.json#L16

**Comment by mauveine** [comment]
You can also modify the migrations/.common.js and change the apiVersion to 6.6 or some other compatible version.

**Comment by dimasch** [comment]
> You can also modify the migrations/.common.js and change the apiVersion to 6.6 or some other compatible version.

Thanks, @mauveine 
You tested your workaround?

This not related to versions defference, we have the same ES version and the same in ES client connection.

Have a good day!

**Comment by pkarw** [comment]
Fixed with #184
